source/slang/emit.cpp

```diff
@@ -3599,34 +3601,58 @@ struct EmitVisitor
         }
     }
 
-    void emitComparison(EmitContext* ctx, IRInst* inst, IREmitMode mode, EOpInfo& inOutOuterPrec, const EOpInfo& opPrec, bool* needCloseOut)
+    void _maybeEmitGLSLVectorCast(EmitContext* ctx, IRType* vecType, IRInst* inst, IREmitMode mode, const EOpInfo& opPrec)
```

Comment:
`opPrec` here should probably be `inOuterPrec` and be copied to a local `outerPrec` as we use in other cases.

Reply:
(This probably doesn't matter much since it would only be called with `kEOp_General` anyway)